akka-docs/src/main/paradox/general/message-delivery-reliability.md

```diff
@@ -202,14 +201,14 @@ actually do apply the best effort to keep our tests stable. A local `tell`
 operation can however fail for the same reasons as a normal method call can on
 the JVM:
 
- * `StackOverflowError`
- * `OutOfMemoryError`
- * other `VirtualMachineError`
+* `StackOverflowError`
+* `OutOfMemoryError`
+* other `VirtualMachineError`
```

Comment:
Isn't the space before bullet required (or am I mixing up markup languages again)?

Reply:
My linter says, no space before a list. Checked locally and it rendered fine

Reply:
0 spaces first level, 2 spaces second level and so on. I think this is how ot woks. But interpreters are very lax on those mistakes.

Like the trailing space, it's 0 or 2.